uk.co.compucorp.civicrm.tasksassignments/js/test/controllers/modal/modal-document_test.js

```diff
@@ -198,6 +204,57 @@ define([
       });
     });
 
+    describe('$scope.onContactChanged', function () {
+      beforeEach(function () {
+        spyOn(ContactService, 'updateCache').and.returnValue({});
+        spyOn(AssignmentService, 'search').and.returnValue($q.resolve([]));
+      });
+
+      beforeEach(function () {
+        initController();
+        $scope.onContactChanged(contactMock.contact);
+      });
+
+      it('calls contact service to cache selected Contact', function () {
+        expect(ContactService.updateCache).toHaveBeenCalled();
```

Comment:
I am testing `ContactService.updateCache` as this will will be called when we call `$scope.cacheContact(selectedContact)`.